### ISSUE TITLE
Gutenberg: 9.6.x Plugin Upgrade

### ISSUE THREAD

**Issue by fullofcaffeine** [Automattic/wp-calypso#48426]
Previous upgrade: https://github.com/Automattic/wp-calypso/issues/47881

### Process

- [x] Install and activate
  - [x] Copy 9.5.2 to 9.6.0 (D54393-code)
  - [x] Install 9.6.0 RC1 on simple sites (D54396-code)
  - [x] Activate 9.6.0 RC1 on edge sites (D54397-code)
  - [x] Copy 9.6.0 to 9.6.1 (D54745-code) _(ended up not being activated as 9.6.2 was release shortly thereafter)_
  - [x] Copy 9.6.0 to 9.6.2 (D54750-code)
  - [x] Install 9.6.2 (D54750-code)
  - [x] Activate 9.6.2 on edge sites (D54746-code)
  - [x] Activate 9.6.2 on production sites (D54465-code)
  - [x] Request update for Atomic sites (p9o2xV-1cR-p2)
- [x] Testing
  - [x] Calypso e2e (https://github.com/Automattic/wp-calypso/pull/48462)
  - [x] TC (D54465-code, D55104-code)
- [x] Publish internal announcements (p7DVsv-aaG-p2, p1610379951103100-slack-CB0B2G43X)
- [x] Clean up unused releases
   - [x] 9.1.1 (D54335-code)
   - [x] 9.2.2 (D54336-code)
   - [x] 9.3.0 (D54337-code)
   - [x] 9.4.0 (D54338-code)
   - [x] 9.4.1 (D54339-code) _(Last release on production. Remove only after 9.6 is released to prod!)_
   - [x] 9.4.2 (D54340-code)
   - [x] 9.5.0 (D54341-code)
   - [x] 9.6.0 (D54911-code)
   - [x] 9.6.1 (D54913-code)
   - [x] 9.5.1 (D54915-code)
- [x] Update the Log of Update Attempts [pcoGjb-u-p2] and list patches applied (pcoGjb-A-p2)
- [x] Open a new issue for the next upgrade, transfer remaining tasks, close this issue.

### Issues / Blockers 🐛 

- [x] Homepage WSOD when gutenberg-edge sticker applied (#48733, fixed via D55104-code)

**Comment by WunderBart** [comment]
Next upgrade: https://github.com/Automattic/wp-calypso/issues/48771
